generated/nidigital/nidigital/session.py

```diff
@@ -2970,8 +3019,8 @@ def is_done(self):
         return bool(done_ctype.value)
 
     @ivi_synchronized
-    def load_levels(self, levels_file_path):
-        r'''load_levels
+    def _load_levels(self, levels_file_path):
```

Comment:
The parameter names of the public function are different:
specifications_file_paths, levels_file_paths, timing_file_paths

Please be consistent.

Reply:
There's an issue - #1393 to address the parameter names of these methods. The thought was to make the parameter names be just `file_path` since the method name implies the kind of file and since only one file can be loaded at a time.